draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -530,7 +530,9 @@ data in a cookie-value SHOULD encode that data, for example, using Base64
 {{RFC4648}}.
 
 The domain-value is a subdomain as defined by {{RFC1034}}, Section 3.5, and
-as enhanced by {{RFC1123}}, Section 2.1.
+as enhanced by {{RFC1123}}, Section 2.1. The domain-value MUST be a string of
+{{USASCII}} characters, such as one obtained by applying the "ToASCII" operation
+defined in {{Section 4 of RFC3490}}.
```

Comment:
I think this is redundant with how subdomain is defined in the referenced RFCs so I'd not include this. (If you really want to include it I'd make it a statement of fact. ("Thus, the domain-value is a string of {{USASCII}} characters, such as one ...")

Reply:
I think that's a good suggestion! I might be reading the referenced RFCs wrong but for me they operate more under an assumption of ASCII than strictly requiring it? In any case, I do think it's worthwhile calling this out in the server section so I'll go with what you suggested.